Pod/Classes/RZMessagingWindow.m

```diff
@@ -292,28 +292,48 @@ @implementation RZRootMessagingViewController
 
 #pragma mark - Orientation method overrides
 
--(BOOL)shouldAutorotate
+-(UIStatusBarStyle)preferredStatusBarStyle
 {
     UIViewController *topViewController = [RZRootMessagingViewController topViewController];
+
+    UIStatusBarStyle statusBarStyle;
     
-    if ( topViewController == self ) {
-        return [super shouldAutorotate];
+    if ( [(RZMessagingWindow *)self.view.window errorIsBeingPresented] ) {
+        statusBarStyle = [super preferredStatusBarStyle];
     }
     else {
-        return topViewController.shouldAutorotate;
+        statusBarStyle = topViewController.preferredStatusBarStyle;
```

Comment:
`preferredStatusBarStyle` isn't a property so shouldn't use dot notation. @ZevEisenberg That's our style right?